src/app/clients/clientsTable/format.ts

```diff
@@ -1,16 +1,18 @@
-import { DeleteClientError } from "../deleteClient";
-import { IsClientActiveError } from "../getExpandedClientDetails";
-
-export const getIsClientActiveErrorMessage = (error: IsClientActiveError): string => {
-    switch (error.type) {
-        case "failedClientIsActiveFetch":
-            return `Failed to determine whether client is active. Please reload. Log ID: ${error.logId}`;
-    }
-};
-
-export const getDeleteClientErrorMessage = (error: DeleteClientError): string => {
-    switch (error.type) {
-        case "failedClientDeletion":
-            return `Failed to delete client. Please try again later. Log ID: ${error.logId}`;
-    }
-};
+import { DeleteClientError } from "../deleteClient";
+import { IsClientActiveError } from "../getExpandedClientDetails";
+
+export const getIsClientActiveErrorMessage = (error: IsClientActiveError): string => {
+    switch (error.type) {
+        case "failedClientIsActiveFetch":
+            return `Failed to determine whether client is active. Please reload. Log ID: ${error.logId}`;
+    }
+};
+
+export const getDeleteClientErrorMessage = (error: DeleteClientError): string => {
+    switch (error.type) {
+        case "failedClientDeletion":
+            return `Failed to delete client. Please try again later. Log ID: ${error.logId}`;
+        case "eventInsertionFailed":

```

Comment:
Think you need to update DeleteClientErrorType to include this new error

Reply:
This commit acceidentally excluded the Type update as the file wasn't saved locally, it should be included now.